WhoIsThat/WhoIsThat/Connections/RestService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WhoIsThat.Models;
+
+namespace WhoIsThat.Connections
+{
+    public class RestService
+    {
+        HttpClient Client { get; set; }
+
+        public RestService()
+        {
+            Client = new HttpClient();
+        }
+
+        /// <summary>
+        /// Calls ImageObjectElementController in backend and returns list of image objects
+        /// </summary>
+        /// <returns>List of ImageObject instances</returns>
+        public async Task<List<ImageObject>> GetImageObjects()
+        {
+            string restUrl = "https://whoisthatserverstorage2.azurewebsites.net/api/images/all";
```

Comment:
You should store endpoints in a config file of sorts. You could possibly pass it as a parameter or at least set is as `RestService` base